security/pkg/pki/util/keycertbundle_test.go

```diff
@@ -343,145 +343,131 @@ func TestNewVerifiedKeyCertBundleFromFile(t *testing.T) {
 
 // Test the root cert expiry timestamp can be extracted correctly.
 func TestExtractRootCertExpiryTimestamp(t *testing.T) {
```

Comment:
These tests weren't great because it would call ExtractRootCertExpiryTimestamp on the same keybundle in every test case.